ci/cpu/libcuml/build_libcuml.sh

```diff
@@ -10,6 +10,10 @@ if [ "$BUILD_LIBCUML" == '1' -o "$BUILD_CUML" == '1' ]; then
     CUDA_REL=${CUDA:0:4}
   fi
 
+  # for libclang installation
+  # without this, the below build used to fail while trying to find libclang!
+  conda config --add channels rapidsai
```

Comment:
You can remove this now. I've added labels to fix this. And a more permanent fix is here: https://github.com/rapidsai/gpuci-build-environment/pull/25

Reply:
Thanks @raydouglass . Will remove this once that PR gets merged. Or else, it seems to cause CI failures.

Reply:
Done.